### DISCUSSION TITLE
Open an note and navigate back selects allways the same (last) project

### DISCUSSION THREAD

**Discussion by harryinet** [marcusolsson/obsidian-projects#795] [General]
From Table View an others, if i open an note by name-link und than press navigate back i am not in the project of this last opend note. I land in the last of the project combo box.
Okay, i can open a separate tab with CTRL an Click, what i write would be a "nice to have" ...

**Comment by mitchellwrosen** [comment]
Agreed, I find the projects boards rather hard to find and navigate. Is there a better way than just running the "Projects: Show projects" command over and over, always using the drop-down to pick the project I want to see?

It would be great to have an Obsidian note wrapper around a project, or something, so I can just go Search -> Start typing project name -> Open
